pkg/agent/agent_linux.go

```diff
@@ -312,3 +313,12 @@ func (i *Initializer) RestoreOVSBridge() {
 func (i *Initializer) setInterfaceMTU(iface string, mtu int) error {
 	return i.ovsBridgeClient.SetInterfaceMTU(iface, mtu)
 }
+
+func (i *Initializer) setTXChecksumOffload() error {
```

Comment:
Maybe name it as `disableTXChecksumOffload`?
You mentioned `when changing disableTXChecksumOffload from true back
to false, TX checksum offload of Antrea gateway will not be enabled
automatically,` If user changes `disableTXChecksumOffload` in the config file, I assume Antrea agent need to be restarted? then I think we can check and enable it again automatically in codes?

Btw, you need re-base the codes and add more test cases for code coverage improvement.

Reply:
> Maybe name it as disableTXChecksumOffload?

There is variable of Initializer which is named `disableTXChecksumOffload`

> Btw, you need re-base the codes and add more test cases for code coverage improvement.

Rebased, and could you provide some examples about such test?